src/Query/Filter/Field.ts

```diff
@@ -309,4 +310,12 @@ export abstract class Field {
     public createReverseGrouper(): Grouper {
         return this.createGrouper(true);
     }
+
+    protected defaultComparator: Comparator = (a: Task, b: Task) => {
+        const groupA = this.grouper()(a)[0];
+        const groupB = this.grouper()(b)[0];
```

Comment:
What happens when you test this on a field that does not support grouping?

Reply:
Like description, for example. But better to create a fake field in the test that is guaranteed to not support grouping.